webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -75,6 +77,10 @@ export default class YoutubeVideo extends React.PureComponent<Props, State> {
                     height='360px'
                     frameBorder='0'
                     allowFullScreen={true}
+                    allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share'
+                    referrerPolicy={this.props.youtubeReferrerPolicy ? 'strict-origin-when-cross-origin' : undefined}
```

Comment:
shouldn't we better set this directly to 'no-referrer' ?

Reply:
I chose to use it indirectly to avoid having to find and change these parts one by one when the `Referrer-Policy` set globally changes. If 'Referrer-Policy' does not change no matter what, I will change it from `undefined` to `no-referrer`

Reply:
it shouldn't change. Even for clarity starting it with `no-referrer` works better for me.

Reply:
I understood that YouTube should remain 'no-ferrer' even if the global settings change. I'll change it.